user_docs/en/userGuide.md

```diff
@@ -3656,9 +3657,7 @@ For more information, read the in-depth section: [Add-ons and the Add-on Store](
 
 ### Create portable copy {#CreatePortableCopy}
```

Comment:
Before 2024.3beta1, this paragraph was mentioning more or less the case where NVDA can be installed from the tools menu of a portable (or temporary) copy.

This is now less the case, and actually all the references to NVDA's installation are being removed from this paragraph in this PR.

To make things clear, we should create a dedicated paragraph "Install NVDA".to describe this item from the Tools submenu, with a link to the detailed paragraph about installation. Can it be accepted here or should I do it in another PR against master (i.e. for 2024.4)?